tox.ini

```diff
@@ -15,7 +15,7 @@ setenv=
     pyqt5: PYTEST_QT_API=pyqt5
     QT_QPA_PLATFORM=offscreen
 
-passenv=DISPLAY XAUTHORITY USER USERNAME
+passenv=DISPLAY XAUTHORITY USER USERNAME COLUMNS
```

Comment:
Unrelated, but I typically use `COLUMN=80 tox -e ...` when I want copyable pytest-output, like for #338.